jetkit/test/test_auth.py

```diff
@@ -87,3 +88,29 @@ def test_sign_up(client_unauthenticated, api_auth, client):
     )
 
     assert not sign_up_response == 200
+
+
+def test_validate_email():
+    test_email = "me@mail.com"
+
+    assert validate_email(test_email, allowed_domains=["mail.com"], allowed_emails=None)
```

Comment:
Looks like a usecase for pytest's `parametrize` (https://docs.pytest.org/en/2.8.7/parametrize.html)

Reply:
nice. fixed